Content.Server/Power/Components/CablePlacerComponent.cs

```diff
@@ -1,16 +1,38 @@
 using Robust.Shared.Prototypes;
 using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
 using Content.Shared.Power;
+using Content.Shared.Whitelist;
 
 namespace Content.Server.Power.Components
 {
     [RegisterComponent]
     public sealed partial class CablePlacerComponent : Component
     {
+        /// <summary>
+        /// The structure prototype for the cable coil to place.
+        /// </summary>
         [DataField("cablePrototypeID", customTypeSerializer:typeof(PrototypeIdSerializer<EntityPrototype>))]
         public string? CablePrototypeId = "CableHV";
 
-        [DataField("blockingWireType")]
+        /// <summary>
+        /// What kind of wire prevents placing this wire over it as CableType.
+        /// </summary>
+        [DataField]
         public CableType BlockingCableType = CableType.HighVoltage;
```

Comment:
You didn't update any of the old datafields for this change for this so they will break.

Reply:
I decided to leave it as wire so no breaky downstream from this.